File: 프로그래머스/0/181942. 문자열 섞기/문자열 섞기.java
class Solution {
    public String solution(String str1, String str2) {
    	String answer = "";
    	for(int cnt = 0; cnt < str1.length(); cnt++) {
    		answer += str1.charAt(cnt);
    		answer += str2.charAt(cnt);
    	}
    	
        return answer;
        }
    }

File: 프로그래머스/0/181942. 문자열 섞기/문자열 섞기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

char* solution(const char* str1, const char* str2) {
    int len = strlen(str1) + strlen(str2);
    char* answer = (char*)malloc(len + 1);
    
    int str1_len = 0;
    int str2_len = 0;
    
    for(int i = 0; i < len; i++){
        if(i % 2){
            answer[i] = str2[str2_len];
            str2_len++;
        }else {
            answer[i] = str1[str1_len];
            str1_len++;
        }
    }
    
    answer[len] = '\0';
    
    return answer;
}